### PR TITLE
fix: jans-cli README broken link

### PR DESCRIPTION
**Pull request by devrimyatar**
* Remove broken link
* Add Documentation section to point CLI docs

### CODE REVIEW

File: jans-cli/README.md [hunk 1]
```diff
@@ -141,5 +144,9 @@ Now selecting 1 and it returns our desired result as below image:
 
 ![default-authentication-method.png](../docs/assets/user/using-jans-cli/images/image-im-cur-default-auth-03042021.png)
 
-So, That was a quick start to view how this _jans-cli_ Interactive Mode works. Please, follow this [link](../docs/user/using-jans-cli) to read the _jans-cli_ docs for a better understanding of the Janssen Command-Line.
+So, That was a quick start to view how this _jans-cli_ Interactive Mode works. Please, see documentation for further information.
+
+
 
+# _Documentation_
+We have extensive documentation for CLI: [https://github.com/JanssenProject/jans/tree/main/docs/admin/jans-cli](https://github.com/JanssenProject/jans/tree/main/docs/admin/jans-cli)
```
Comment (moabu) [inline jans-cli/README.md:152]: replace with "Extensive documentation for the CLI can be found [here](https://github.com/JanssenProject/jans/tree/main/docs/admin/jans-cli)"

File: jans-cli/README.md [hunk 2]
```diff
@@ -141,5 +144,9 @@ Now selecting 1 and it returns our desired result as below image:
 
 ![default-authentication-method.png](../docs/assets/user/using-jans-cli/images/image-im-cur-default-auth-03042021.png)
 
-So, That was a quick start to view how this _jans-cli_ Interactive Mode works. Please, follow this [link](../docs/user/using-jans-cli) to read the _jans-cli_ docs for a better understanding of the Janssen Command-Line.
+So, That was a quick start to view how this _jans-cli_ Interactive Mode works. Please, see documentation for further information.
```
Comment (moabu) [inline jans-cli/README.md:147]: Replace with "The above was a quick start to how _jans-cli_ Interactive Mode works. Please, see documentation for further information."

**Review by moabu — CHANGES_REQUESTED** [review]

**Comment by ossdhaval** [comment]
Hi @devrimyatar 
This is a quick one. Merge it if it is still relevant, else close it.

**Comment by ossdhaval** [comment]
Closing this PR as confirmed by @devrimyatar to be outdated.

**Pull request closed without merging**